### ISSUE TITLE
missed func: this._renderOverlay in AppNavigator.js (v. 5.2.1)

### ISSUE THREAD

**Issue by alexicum** [start-react/native-starter-kit#49]
```
<NavigationCardStack
  navigationState={this.props.navigation}
  renderOverlay={this._renderOverlay}
  renderScene={this._renderScene}
/>
```

**Comment by SupriyaKalghatgi** [comment]
@alexicum `this._renderOverlay` is not defined

This would be updated with next release

**Comment by alexicum** [comment]
@SupriyaKalghatgi 

Found discussion about renderOverlay (but may be it is not related):
https://github.com/facebook/react-native/issues/7210#issuecomment-227317718

> This issue was addressed in facebook/react-native@3a62314. Both renderScene and renderOverlay are now deprecated. NavigationTransitioner now delegates rendering of all its children to a user provided render function in which you can place your overlay however you want.

**Comment by SupriyaKalghatgi** [comment]
@alexicum Thank you 

Probably NSK will exclude this in upcoming release
